yarn-project/cli/src/index.ts

```diff
@@ -218,14 +218,16 @@ export function getProgram(log: LogFn, debugLogger: DebugLogger): Command {
       const args = encodeArgs(rawArgs, constructorArtifact!.parameters);
       debugLogger(`Encoded arguments: ${args.join(', ')}`);
 
-      const tx = deployer.deploy(...args).send({ contractAddressSalt: salt });
+      const deployMethod = deployer.deploy(...args);
+      const tx = deployMethod.send({ contractAddressSalt: salt });
       const txHash = await tx.getTxHash();
       debugLogger(`Deploy tx sent with hash ${txHash}`);
       if (wait) {
         const deployed = await tx.wait();
         log(`\nContract deployed at ${deployed.contractAddress!.toString()}\n`);
       } else {
-        log(`\nDeployment transaction hash: ${txHash}\n`);
+        log(`\nContract Address: ${deployMethod.completeAddress?.address.toString() ?? 'N/A'}`);
```

Comment:
This API kind of sucks, you shouldn't need to keep the `deployMethod` around just to get the deployment address. Maybe we should inject it into `yarn-project/aztec.js/src/contract_deployer/deploy_sent_tx.ts` so you can query it from the `tx` object directly?

(not a note for this PR, but more of a comment for something we need to change elsewhere)

Reply:
Yeah I thought it was a bit rubbish. I went ahead and made the change.